rflx/pyrflx/typevalue.py

```diff
@@ -705,6 +726,25 @@ def set_refinement(fld: MessageValue.Field, fld_name: str) -> None:
                     ):
                         fld.typeval.set_refinement(ref.sdu, self._refinements)
 
+        def check_outgoing_condition_satisfied() -> None:
+            if all(
+                [
+                    self.__simplified(o.condition) == FALSE
+                    for o in self._type.outgoing(Field(field_name))
+                ]
+            ):
+                self._fields[field_name].typeval.clear()
+                if isinstance(value, bytes):
+                    value_repr = "x" + value.hex()
```

Comment:
Why is it `"x" + ...` and not `"0x" + ...`?

Reply:
removed this line as discussed